.github/workflows/lint.yaml

```diff
@@ -9,6 +9,7 @@ on:
       - development
       - master
       - release
+      - rnn
```

Comment:
`rnn` branch will be deleted -> delete
`release` branch not used anymore in workflow -> delete